book/models.py

```diff
@@ -8,11 +8,22 @@ class Category(models.Model):
     def __str__(self):
         return self.name
 
+
+class Author(models.Model):
+    name = models.CharField(max_length = 50)
```

Comment:
```suggestion
    name = models.CharField(max_length=50)
```
コーディング規約(pep8) では、スペースはいらない